core/templates/I18nFooter.ts

```diff
@@ -50,7 +52,16 @@ angular.module('oppia').directive('i18nFooter', [
           };
           ctrl.$onInit = function() {
             ctrl.supportedSiteLanguages = SUPPORTED_SITE_LANGUAGES;
-
+            // Fetching the saved language code from cookie as $translate in
+            //  angularjs uses cookies to store the last used code and we need
```

Comment:
drop space before angularjs

add "to" after "need"

Reply:
Done!